### ISSUE TITLE
[FEAT] Pass the full resource to the `accessControlProvider` `can` method

### ISSUE THREAD

**Issue by alexolivier** [pankod/refine#2220]
**Is your feature request related to a problem? Please describe.**
When making use of the `accessControlProvider` for authorization, it would be useful to have access to the full resource object. This will enable Attribute Based Access Control (ABAC), for example granting permissions based on the value of a field in the resource object.

**Describe the solution you'd like**
In the `can` function, pass in the full object as a parameter, rather than just the ID.
```
<Refine
      accessControlProvider={{
        can: async ({ action, params, resource }) => {
```

Users can then make decisions based on any field in the resource.

**Describe alternatives you've considered**
You could do a separate call to get the resource again from the backend but that is going to double the load on your service.

**Additional context**
Any ABAC supporting authorization system, such as [Cerbos](https://cerbos.dev), is going to need this.

**Comment by omeraplak** [comment]
Hey @alexolivier 👋,
Thanks for contacting us. We discussed this with the team, we decided to add the `meta` property. We will update you soon!

**Comment by omeraplak** [comment]
Hey @alexolivier ,
We've released [`@pankod/refine-react-router-v6@3.29.0`](https://github.com/pankod/refine/releases/tag/%40pankod%2Frefine-react-router-v6%403.29.0), [`@pankod/refine-react-location@3.29.0`](https://github.com/pankod/refine/releases/tag/%40pankod%2Frefine-react-location%403.29.0), [`@pankod/refine-nextjs-router@3.29.0`](https://github.com/pankod/refine/releases/tag/%40pankod%2Frefine-nextjs-router%403.29.0), [`@pankod/refine-core@3.50.0`](https://github.com/pankod/refine/releases/tag/%40pankod%2Frefine-core%403.50.0) and [`@pankod/refine-kbar@0.6.0`](https://github.com/pankod/refine/releases/tag/%40pankod%2Frefine-kbar%400.6.0) 💥
